sql/npc_list.sql

```diff
@@ -31423,11 +31423,11 @@ INSERT INTO `npc_list` VALUES (17801264,'2','2',213,-29.184,-2.992,89.212,0,40,4
 INSERT INTO `npc_list` VALUES (17801265,'3','3',205,-20.758,0.398,82.678,0,40,40,0,0,0,6,27,0x0000A00100000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17801266,'4','4',142,-13.865,-0.443,101.201,0,40,40,0,0,0,6,27,0x0000A00100000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17801267,'5','5',153,-13.147,-0.482,98.979,0,40,40,0,0,0,6,27,0x0000A00100000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17801268,'Magriffon','Magriffon',159,30.828,0.214,104.373,0,40,40,0,0,0,6,25,0x01000D0300104820483048404850C1602D700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17801269,'Kihujin','Kihujin',199,10.050,0.885,100.947,0,40,24,0,0,0,6,27,0x00006A0000000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17801270,'Kaizoku01','Kaizoku01',170,11.444,0.102,91.730,0,40,40,0,0,0,6,27,0x010008011A102A200D3004400D501D6129700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17801271,'Kaizoku02','Kaizoku02',171,12.052,0.000,89.438,0,40,40,0,0,0,6,27,0x01000C011A102A200D3004400D50336100700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17801272,'Kaizoku03','Kaizoku03',158,14.220,-0.059,88.892,0,40,40,0,0,0,6,27,0x010007011A102A200D3004400D50B1602A700000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17801268,'Magriffon','Magriffon',159,30.828,0.214,104.373,0,50,50,0,0,0,6,25,0x01000D0300104820483048404850C1602D700000,32,NULL,1);
```

Comment:
these were all moving and animating as 50 in caps? that's actually not typical for pathing entities

Reply:
Also, 1268 is the event NPC, not the one visible (that's 1262)

Reply:
The reason I ask is because at the time people in dsp _assumed_ it all went from 40 to 50, and in reality some were never 40 either and some of the 40s stayed 40. With NPCs that walk around they are usually much slower than 40 unless running.

I decided to check him myself. He's not moving at 50. he animates at 50 but speed is far less: 
![image](https://github.com/LandSandBoat/server/assets/6871475/8b081e05-77ae-4347-a65b-9a7b567b5858)

_but wait Teo!_ you might be thinking _how the F is it 13.000001 ?_

Well the answer is in packet you see the raw base value, but in client memory you see a float after modifications such as gravity quickening flee etc. So this is the first NPC I've ever seen that had anything after the decimal place. **I see if often enough on mobs and players, but Magriffon here implies NPCs can have speed modifications. _That is a new discovery!_** So presumably he spawns in at 50, and that's in the packet. and then some weight effect is applied to take him way down. You can tell just watching him pace back and forth that ain't no 50.